pulp_2_tests/tests/rpm/api_v2/test_modularity.py

```diff
@@ -31,17 +37,29 @@ def setUpClass(cls):
         if cls.cfg.pulp_version < Version('2.17'):
             raise unittest.SkipTest('This test requires Pulp 2.17 or newer.')
         cls.client = api.Client(cls.cfg, api.json_handler)
+        cls.repos = []
+        try:
+            for _ in range(2):
+                body = gen_repo()
+                if _ == 0:  # adding feed only to first repo
```

Comment:
If you need to use the index to verify a certain value, another strategy should be used instead of  `_`.